packages/ffe-core-react/src/typography/LinkText.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { bool, node, string } from 'prop-types';
+import classNames from 'classnames';
+
+const LinkText = (props) => {
+    const {
+        children,
+        className,
+        href,
+        underline,
+        ...rest
```

Comment:
I would love for this to have an `element` prop, so that I can use it with `react-router`-s `Link` component.

Reply:
That also makes the `href` prop unvalidatable - because you don't know whether or not we need a `href` prop or a `to` prop.